managed/src/main/java/com/yugabyte/yw/common/PlacementInfoUtil.java

```diff
@@ -2300,14 +2307,46 @@ public static Map<String, String> getConfigPerNamespace(
     return namespaceToConfig;
   }
 
+  /**
+   * Returns kubeconfig for each Pod's FQDN from the nodeDetailsSet. This method is useful for both
```

Comment:
minor: Returns a map `pod_fqdn -> KUBECONFIG string` for all pods in nodeDetailSet